pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,36 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+  - build
+  - -t
+  - gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container
+  - build/x86_64_xenial
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+  - run
+  - -v
+  - /workspace:/workspace
+  - -w
+  - /workspace/src/
+  - gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container
+  - /bin/bash
+  - -c
+  - make purge && make -j4 DEB_PKG=stackdriver-metadata.deb DISTRO=xenial deb
```

Comment:
Should this be quoted?

Reply:
It seems to work without the quotes, I don't have a link to best practices one way or another, so I'll leave it as-is for now.